src/DynamoCoreWpf/Views/Core/NodeView.xaml.cs

```diff
@@ -686,7 +686,7 @@ private void OnNodeViewMouseMove(object sender, MouseEventArgs e)
         /// So we can't use MouseLeave/MouseEnter events.
         /// In this case, when we want to ensure, that mouse really left node, we use HitTest.
         /// </summary>
-        /// <param name="mousePosition">Correct position of mouse</param>
+        /// <param name="mousePosition">Currect position of mouse</param>
```

Comment:
We can probably avoid this line rollback :)

Reply:
We roll back all or nothing! lol. I can change that.